<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editworkers</class>
 <widget class="QDialog" name="editworkers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="managers_tab_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>661</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="workers_tab_basicinfo">
    <attribute name="title">
     <string>基本信息</string>
    </attribute>
    <widget class="QPushButton" name="managers_okpushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>300</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="workers_table_usergroupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>531</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>用户信息</string>
     </property>
     <widget class="QLabel" name="managers_idlabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>工号</string>
      </property>
      <property name="buddy">
       <cstring>managers_idlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_idlineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="managers_namelabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
      <property name="buddy">
       <cstring>managers_namelineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_namelineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="workers_table_basicgroupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>551</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>基本信息</string>
     </property>
     <widget class="QLabel" name="managers_sexlabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
      <property name="buddy">
       <cstring>managers_sexlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_agelineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="managers_personnellabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>所属部门</string>
      </property>
      <property name="buddy">
       <cstring>managers_personnellineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="managers_phonelabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>联系方式</string>
      </property>
      <property name="buddy">
       <cstring>managers_phonelineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="managers_positionlabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>职称</string>
      </property>
      <property name="buddy">
       <cstring>managers_positionlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="managers_addresslabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>籍贯</string>
      </property>
      <property name="buddy">
       <cstring>managers_addresslineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="managers_idcardlabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>身份证号</string>
      </property>
      <property name="buddy">
       <cstring>managers_idcardlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="managers_birthlabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>出生年月</string>
      </property>
      <property name="buddy">
       <cstring>managers_birthlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_addresslineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_positionlineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_personnellineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="managers_agelabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄</string>
      </property>
      <property name="buddy">
       <cstring>managers_agelineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_phonelineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_birthlineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_idcardlineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="managers_sexlineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="managers_closepushButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="managers_againpushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>300</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="workers_tab_payinfo">
    <attribute name="title">
     <string>工资信息</string>
    </attribute>
    <widget class="QDateEdit" name="managers_dateEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-M</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="managers_datelabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
    <widget class="QLabel" name="managers_totalpaylabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>总工资 </string>
     </property>
    </widget>
    <widget class="QLabel" name="managers_taxlabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>税收</string>
     </property>
    </widget>
    <widget class="QLabel" name="managers_incomelabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>净工资</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="managers_totalpaylineEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="managers_taxlineEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="managers_incomelineEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="managers_okpushButton_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>300</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="managers_closepushButton_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="workers_tab_attendinfo">
    <attribute name="title">
     <string>考勤信息</string>
    </attribute>
    <widget class="QDateEdit" name="managers_dateEdit_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2013</year>
       <month>5</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2013</year>
       <month>5</month>
       <day>1</day>
      </date>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy-M</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="managers_datelabel_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
    <widget class="QLabel" name="managers_latelabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>迟到次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="managers_absentlabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>缺席次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="managers_leavelabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>请假次数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="managers_latelineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="managers_leavelineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="managers_absentlineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="managers_okpushButton_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>300</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="managers_closepushButton_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>300</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="managers_firstpushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;First</string>
   </property>
  </widget>
  <widget class="QPushButton" name="managers_previouspushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="managers_nextpushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="managers_lastpushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Last&gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
